index.js

```diff
@@ -41,18 +55,22 @@ function fastatic(options) {
 	// 3. revision files
 	// ...
 	// 4. copy to final dest
-	.then(() => copy(config.temp, config.dest))
+	// .then(() => copy(config.temp, config.dest))
 	// 5. remove temp dir
 	.catch(err => console.log('error', err)) // if anything goes wrong, we skip all steps, catch errors and remove temp
 	//.then(() => remove(config.temp))
 	// 6. create stats
+	.then(() => stats(config))
```

Comment:
`stats()` should run before copying `copy(config.temp, config.dest)` because if `config.src` and `config.dist` are the same, so will `config.src` and `config.temp` be at this point. Meaning stats will show 0%.